packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -142,6 +142,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
   private _processingTouch: boolean;
   private _lastTouchTimeoutId: number | undefined;
   private _processingExpandCollapseKeyOnly: boolean;
+  private _focusInputAfterBlur: boolean;
```

Comment:
Seems like this variable mimics this.state.focused or if focusInputAfterClose is false.  It seems like there's a lot of variables tracking more or less the same thing, is there any way we can simplify this?

Reply:
I actually think we can use the focused state to mimic this and am currently looking to see how I can use that.